src/components/GordonOffline/index.jsx

```diff
@@ -0,0 +1,72 @@
244.750 267.317,244.200 266.800 C 243.650 266.283,243.200 265.695,243.200 265.494 C 243.200 264.478,234.027 254.800,233.064 254.800 C 232.776 254.800,232.117 254.350,231.600 253.800 C 231.083 253.250,230.450 252.800,230.192 252.800 C 229.935 252.800,229.414 252.458,229.036 252.040 C 228.657 251.621,227.734 251.164,226.985 251.024 C 226.235 250.883,225.482 250.541,225.311 250.264 C 225.140 249.987,224.190 249.526,223.200 249.240 C 222.210 248.955,221.130 248.493,220.800 248.214 C 220.470 247.935,219.152 247.589,217.870 247.446 C 216.589 247.303,214.879 246.909,214.070 246.570 C 212.352 245.850,204.466 245.483,201.400 245.982 M214.296 258.400 C 215.116 258.840,216.203 259.200,216.711 259.200 C 217.967 259.200,224.576 262.472,225.935 263.767 C 230.153 267.787,233.200 270.885,233.200 271.153 C 233.200 271.335,233.650 271.969,234.200 272.562 C 234.750 273.156,235.200 273.983,235.200 274.401 C 235.200 274.818,235.482 275.318,235.827 275.511 C 236.172 275.704,236.632 276.655,236.850 277.625 C 237.068 278.595,237.490 279.661,237.788 279.994 C 239.652 282.079,239.678 298.680,237.822 300.794 C 237.529 301.127,237.159 302.030,237.000 302.800 C 236.841 303.570,236.484 304.470,236.207 304.800 C 235.929 305.130,235.389 306.075,235.005 306.900 C 234.622 307.725,234.149 308.400,233.954 308.400 C 233.759 308.400,233.600 308.667,233.600 308.993 C 233.600 310.485,227.748 316.030,223.671 318.400 C 223.103 318.730,222.281 319.315,221.844 319.700 C 221.407 320.085,220.699 320.400,220.272 320.400 C 219.844 320.400,218.929 320.750,218.238 321.177 C 217.547 321.604,215.870 322.132,214.512 322.351 C 213.155 322.570,211.686 322.940,211.248 323.174 C 210.068 323.806,202.590 323.716,201.280 323.054 C 200.686 322.754,199.229 322.385,198.042 322.233 C 196.855 322.081,195.235 321.622,194.442 321.212 C 193.649 320.802,192.550 320.266,192.000 320.021 C 190.156 319.200,188.611 318.206,188.081 317.500 C 187.791 317.115,187.317 316.800,187.026 316.800 C 186.283 316.800,180.800 311.648,180.800 310.950 C 180.800 310.634,180.546 310.278,180.237 310.159 C 179.927 310.040,179.460 309.428,179.200 308.800 C 178.940 308.172,178.473 307.560,178.163 307.441 C 177.854 307.322,177.600 306.884,177.600 306.466 C 177.600 306.049,177.229 305.236,176.776 304.660 C 176.323 304.084,175.848 302.935,175.722 302.106 C 175.595 301.278,175.216 300.282,174.878 299.894 C 173.348 298.132,173.401 282.567,174.944 280.862 C 175.305 280.463,175.600 279.590,175.600 278.922 C 175.600 278.254,175.949 277.264,176.375 276.722 C 176.802 276.180,177.271 275.256,177.419 274.668 C 177.566 274.081,177.844 273.600,178.035 273.600 C 178.227 273.600,178.735 272.911,179.165 272.068 C 180.474 269.503,187.456 262.800,188.820 262.800 C 189.233 262.800,189.763 262.442,189.997 262.005 C 190.232 261.567,190.913 261.093,191.512 260.951 C 192.110 260.809,192.883 260.448,193.228 260.150 C 193.573 259.851,194.799 259.428,195.952 259.209 C 197.105 258.990,198.437 258.539,198.913 258.206 C 200.367 257.188,212.315 257.339,214.296 258.400 "
```

Comment:
Because we can't import SVGs in Vite the way we did in React, and SVGs are valid html so this is the simplest solution. You can submit an issue to revise it if you would like. This also doesn't seem like an issue that needs to block releasing.